src/vlan.rs

```diff
@@ -4,11 +4,11 @@ use itertools::Itertools;
 
 fn run_cmd(input: &str) -> Result<i32, i32> {
     println!("{}", input);
-    let split = input.split(" ");
+    let split = input.split(' ');
     let cmd = split.clone().next().unwrap();
     let cmd = std::process::Command::new(cmd).args(split.skip(1)).spawn().unwrap();
```

Comment:
Please note that split by whitespace could be a problem if command contains quotes like `echo '123 456'`